presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingElement.java

```diff
@@ -25,6 +25,8 @@ protected GroupingElement(Optional<NodeLocation> location)
         super(location);
     }
 
+    public abstract List<Expression> getColumns();
```

Comment:
i am not sure this is the best name. `groupingColumns`? keys?

Reply:
Well, it's the columns listed in the grouping element, so kind of obvious, no?